WebDriverAgentLib/Utilities/FBMjpegServer.m

```diff
@@ -102,7 +109,14 @@ - (void)streamScreenshot
     [self scheduleNextScreenshotWithInterval:timerInterval timeStarted:timeStarted];
     return;
   }
+  [self.imageScaler submitImage:screenshotData
```

Comment:
I'd rather only apply scaling if this is explicitly set in settings (either factor or compression are set explicitly). Otherwise it is just going to slow down the process

Reply:
It is only scaled if the scaling factor is not equal to 1: https://github.com/appium/WebDriverAgent/pull/138/files#diff-1bbb80b8f45182835df2d8339409567cR44
But I could add a method `isScalingEnabled` to check it and make it more clear.